mailer.rst

```diff
@@ -783,7 +783,7 @@ environment:
 
 .. code-block:: yaml
 
-    # config/services_dev.yaml
+    # config/services.yaml
```

Comment:
IMO this change is not correct, because you just want to override this service in `dev`, `services_dev.yaml` will only be loaded in `dev`

Reply:
@OskarStark as we spoke about it on Slack, i close this PR.